src/main/resources/db/migration/V1_219_1__add_dependence_and_recovery_new_desired_outcomes_and_rules.sql

```diff
@@ -0,0 +1,192 @@
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'DN547952_6'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'DN547952_7'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'DN547952_8'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'DN547952_9'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_7283_E'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_7283_N'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_7283_S'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_5750'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_5751'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_5752'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_5753'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_5764'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_5776'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_5779'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_5780'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_5781'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_5789'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_5790'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_5791'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_5792'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_5793'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_5794'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_5795'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_5798'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_5799'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_5802'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_5803'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_5806'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_5809'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_5813'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_5815'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_5817'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_5818'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_5819'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_5821'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_5822'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_5823'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_5824'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_5825'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_6553'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_6554'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_6555'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_6556'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_12498_GWENT'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_12498_SOUTHWALES'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_12502_DYFEDPOWYS'),
+       ('e866ff81-49ee-443d-87b6-e14079def942', 'PRJ_12502_NORTHWALES');
```

Comment:
Do we need to add an entry for new contracts which went live last week? PRJ_14249, PRJ_14250, PRJ_14251, PRJ_14252, PRJ_14253, PRJ_14255, PRJ_14403?

Reply:
I've just checked and these are on the spreadsheet for showing new outcomes only, so no match rules are needed.
For dependancy and recovery, it's only women's services that have an extra rule.
Was good to check though. Thanks!